docs/tutorials/send-and-receive-payments.mdx

```diff
@@ -151,7 +151,7 @@ func main () {
 
     // Make sure destination account exists
     destAccountRequest := horizonclient.AccountRequest{AccountID: destination}
-    destinationAccount, err := client.AccountDetail(destAccountRequest)
+    _, err := client.AccountDetail(destAccountRequest)
```

Comment:
I think keeping `destinationAccount` is useful for the context it adds to the example. The variable is never used, but I think the extra clarity makes it worth keeping here.

I'm not very familiar with Go, so preferring `_` may come with some benefits I don't know about. What do you think, @mollykarcher?

Reply:
Whereas declaring but not using a variable is typically a linting concern in most languages, it's a compile error in Go, so this doesn't compile as-written. That said, I agree it's more clear/understandable with a name, and this _is_ just a snippet so one could maybe argue that it doesn't necessarily need to compile, but I personally think it should.

Reply:
Aha!! That's very good to know! I agree that if we're providing it as an example, it should compile and be a useable snippet 👍🏻 

Would it be possible to do something like this to "use up" the `destinationAccount` variable?

```go
destAccountRequest := horizonclient.AccountRequest{AccountID: destination}
destinationAccount, err := client.AccountDetail(destAccountRequest)
_ = destinationAccount
```

After writing that, it looks pretty stupid... Also feels like it introduces a bigger "why" altogether...

Is there a way you can think to write this chunk that might more clearly reflect what the request is doing? Maybe just keeping the old line as a comment with a brief explanation?? 🤔 Don't know if I really like that, either...

@mollykarcher Sorry for the ramblings lol. You know this stuff better than I do, so I'll defer to you: What's the best and clearest approach here?

Reply:
Yeah I'm with you on that feeling weird. Alternatively, we could just add some print/log statements utilizing those accounts. Feels less superfluous and will actually use those variables

Reply:
> Alternatively, we could just add some print/log statements utilizing those accounts. Feels less superfluous and will actually use those variables

How did I miss this comment for 3 weeks!? I'm so sorry!

I like the idea of adding some print statements to the example. Would that be easy to do? Everything else with the PR looks great, and I'd love to get these fixes merged

Reply:
Apparently, embarrassing myself is gonna be my thing today... 🤦🏻‍♂️ Since you've already made those changes, I'll just see myself out lol

Thank you!